keras/wrappers/scikit_learn.py

```diff
@@ -67,15 +91,26 @@ def get_params(self, deep=True):
         Parameters
         ----------
```

Comment:
I'll leave this for afterwards, if you don't mind.
Also, Is there a place where I can see the new convention, other that the code itself?